### PR TITLE
FIX: Strip the Web Client of unused or no longer necessary resources

### PR DESCRIPTION
**Pull request by Akpjunior94**
#### This pull request makes the following changes

* Fixes collabo-community/issue-tickets-ready-for-fixing#24

#

#### General checklist

- [x] My pull request adheres to the [Collabo Community pull request guidelines](https://docs.collabocommunity.com/pull-request-guidelines).
- [x] I have read and understood the **Pull request guideline extension** section found at the bottom of this pull request description/body.
- [x] I have replaced the dummy text under the subheading **Testing checklist** below with the testing checklist from the issue ticket I'm working on.

#

#### Testing checklist

- [x] Styled jsx configurations, and any `.js` files that contain CSS code have been removed.
- [x] Careers and donate pages in code-collabo app, and any files associated to them not used anywhere else in the application have been removed.
- [x] I certify that I ran my checklist

#

#### Pull request guideline extension

> [!NOTE]  
> Do not remove this note or section. This talks about extra things to take note of, in addition to the directions given in the Collabo Community pull request guidelines.

We appreciate the time taken to help tackle issue tickets within the Collabo Community. Please go ahead to submit your pull request even if it is not yet completed, and push to the pull request actively. This will allow us to know that you have not abandoned the issue ticket. However, you should only alert reviewers about your pull request as explained below, to avoid flooding their GitHub notifications unnecessarily.

"Ready for review" implies that things are working as expected, according to how it is specified in the issue ticket you are working on. Ensure that you test the code that you are working on, that it works as expected. Don't just request for review when your pull request is not yet done and working.

The only time you should be making people review something incomplete is: if you want to show them what's not working, so that they can send help or guidance. If this is the case, be explicit about this in our Discord community or in the comment of your pull request (after you have submitted the pull request).

### CODE REVIEW

**Comment by Akpjunior94** [comment]
@Ifycode I want to confirm if there is need to add `package-lock.json `file when there is removal of adding of a dependency.

**Comment by Ifycode** [comment]
Don't add `package-lock.json`, that will be generated on my end later.

File: package.json [hunk 1]
```diff
@@ -13,8 +13,7 @@
   "dependencies": {
     "next": "13.5.4",
     "react": "^18",
-    "react-dom": "^18",
-    "styled-jsx": "^5.1.2"
```
Comment (Ifycode) [inline package.json:17]: babel helps styled-jsx to function. So you also want to remove the babel file at the root of the repo also.

**Review by Ifycode — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Akpjunior94) [reply package.json:17]: removing..

Comment (Akpjunior94) [reply package.json:17]: done

**Review by Ifycode — APPROVED** [review]

**Pull request merged**